### PR TITLE
Add Freezer Monitor (from freezer monitor branch)

### PR DESCRIPTION
**Pull request by charris-msft**
Merging this PR will not automatically deploy Freezer Monitor. There is a subsequent PR coming for azure_arc to update the logonscript with the ability to create configmaps and update the config files.

@sebassem

### CODE REVIEW

File: contoso_supermarket/developer/freezer_monitoring/src/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+version: '3'
+services:
+  mqtt-broker:
+    image: charrisagoracr.azurecr.io/mqtt-broker:latest
```
Comment (sebassem) [inline contoso_supermarket/developer/freezer_monitoring/src/docker-compose.yaml]: does the image have to be provided here ? if yes, let's use placeholders like __acrName__ and there is a GitHub action that I use to replace all of the them with the right values

File: contoso_supermarket/developer/freezer_monitoring/src/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    ports:
+      - "1883:1883"
+  mqtt-simulator:
+    image: charrisagoracr.azurecr.io/mqtt-simulator:latest
```
Comment (sebassem) [inline contoso_supermarket/developer/freezer_monitoring/src/docker-compose.yaml]: does the image have to be provided here ? if yes, let's use placeholders like acrName and there is a GitHub action that I use to replace all of the them with the right values

File: contoso_supermarket/developer/freezer_monitoring/src/mqtt-broker-deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
```
Comment (sebassem) [inline contoso_supermarket/developer/freezer_monitoring/src/mqtt-broker-deployment.yaml]: shouldn't this file be in the operations folder ?

File: contoso_supermarket/developer/freezer_monitoring/src/mqtt-broker-service.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```
Comment (sebassem) [inline contoso_supermarket/developer/freezer_monitoring/src/mqtt-broker-service.yaml]: shouldn't this file be in the operations folder ?

File: contoso_supermarket/developer/freezer_monitoring/src/mqtt-simulator-deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  selector:
+    matchLabels:
+      io.kompose.service: mqtt-simulator
+  template:
+    metadata:
+      labels:
+        io.kompose.service: mqtt-simulator
+    spec:
+      containers:
+        - image: charrisagoracr.azurecr.io/mqtt-simulator:latest
```
Comment (sebassem) [inline contoso_supermarket/developer/freezer_monitoring/src/mqtt-simulator-deployment.yaml]: shouldn't this file be in the operations folder ?

File: contoso_supermarket/operations/freezer_monitoring/Chart.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
+name: sensor-monitor
+description: A helm chart to deploy the Freezer Monitoring application
+home: https://github.com/microsoft/jumpstart-agora-apps
+type: application
+version: 0.1.0
+sources:
+  - https://github.com/charris-msft/jumpstart-agora-apps
```
Comment (sebassem) [inline contoso_supermarket/operations/freezer_monitoring/Chart.yaml]: we can remove the sources property or replace your GitHub Id with microsoft/jumpstart-agora-apps

File: contoso_supermarket/operations/freezer_monitoring/values.yaml [hunk 7]
```diff
@@ -0,0 +1,32 @@
+broker:
+  mqttBroker:
+    image:
+      repository: jumpstartagora.azurecr.io/contoso-supermarket/mqtt-broker
```
Comment (sebassem) [inline contoso_supermarket/operations/freezer_monitoring/values.yaml:4]: we should add a placeholder for acr like _acrName_ and add it in the github action "update-files" to replace it

File: contoso_supermarket/operations/freezer_monitoring/values.yaml [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+    - name: http
+      port: 1883
+      targetPort: 1883
+  replicas: 1
+  type: LoadBalancer
+kubernetesClusterDomain: cluster.local
+simulator:
+  mqttSimulator:
+    image:
+      repository: jumpstartagora.azurecr.io/contoso-supermarket/mqtt-simulator
```
Comment (sebassem) [inline contoso_supermarket/operations/freezer_monitoring/values.yaml:17]: we should add a placeholder for acr like _acrName_ and add it in the github action "update-files" to replace it

File: contoso_supermarket/developer/freezer_monitoring/src/docker-compose.yaml (hunk 1 above)
Comment (charris-msft) [reply contoso_supermarket/developer/freezer_monitoring/src/docker-compose.yaml]: This docker-compose.yaml file is not used. I have removed it.

File: contoso_supermarket/developer/freezer_monitoring/src/mqtt-broker-deployment.yaml (hunk 3 above)
Comment (charris-msft) [reply contoso_supermarket/developer/freezer_monitoring/src/mqtt-broker-deployment.yaml]: it is. this is an older version that I have now removed.

File: contoso_supermarket/developer/freezer_monitoring/src/mqtt-broker-service.yaml (hunk 4 above)
Comment (charris-msft) [reply contoso_supermarket/developer/freezer_monitoring/src/mqtt-broker-service.yaml]: it is. this is an old version that I have now removed.

File: contoso_supermarket/developer/freezer_monitoring/src/mqtt-simulator-deployment.yaml (hunk 5 above)
Comment (charris-msft) [reply contoso_supermarket/developer/freezer_monitoring/src/mqtt-simulator-deployment.yaml]: it is. this is an old version that I have now removed.

File: contoso_supermarket/developer/freezer_monitoring/src/docker-compose.yaml (hunk 2 above)
Comment (charris-msft) [reply contoso_supermarket/developer/freezer_monitoring/src/docker-compose.yaml]: This file is not used and has now been removed.

File: contoso_supermarket/operations/freezer_monitoring/Chart.yaml (hunk 6 above)
Comment (charris-msft) [reply contoso_supermarket/operations/freezer_monitoring/Chart.yaml]: Removing sources - that will be more consistent with contoso_supermarket

File: contoso_supermarket/operations/freezer_monitoring/values.yaml (hunk 7 above)
Comment (charris-msft) [reply contoso_supermarket/operations/freezer_monitoring/values.yaml:4]: looking into this...

File: contoso_supermarket/operations/freezer_monitoring/values.yaml (hunk 8 above)
Comment (charris-msft) [reply contoso_supermarket/operations/freezer_monitoring/values.yaml:17]: looking into this...

File: contoso_supermarket/operations/freezer_monitoring/values.yaml (hunk 7 above)
Comment (charris-msft) [reply contoso_supermarket/operations/freezer_monitoring/values.yaml:4]: These ones are not changing. They will always pull from jumpstartagora.azurecr.io

File: contoso_supermarket/operations/freezer_monitoring/values.yaml (hunk 8 above)
Comment (charris-msft) [reply contoso_supermarket/operations/freezer_monitoring/values.yaml:17]: These ones are not changing. They will always pull from jumpstartagora.azurecr.io

**Comment by charris-msft** [comment]
@sebassem - after all changes, I have deployed again successfully.
Ready for review to continue.

**Review by dkirby-ms — APPROVED** [review]

**Review by dkirby-ms — APPROVED** [review]

**Pull request merged**